.buildkite/scripts/steps/functional/cloud_security_posture_serverless.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/steps/functional/common.sh
+
+export JOB=kibana-cloud-security-posture-serverless-cypress
+export KIBANA_INSTALL_DIR=${KIBANA_BUILD_LOCATION}
+
+echo "--- Cloud Security Posture Workflows Cypress tests on Serverless"
+
+cd x-pack/plugins/security_solution
```

Comment:
This path should be changed to: `x-pack/test/security_solution_cypress` where the package.json commands and the tests live.